package.json

```diff
@@ -51,7 +51,6 @@
     "jshint-stylish": "^1.0.0",
     "merge-stream": "^0.1.7",
     "mocha": "^2.1.0",
-    "napa": "^1.2.0",
```

Comment:
And also remove the `<script src="{{page.include_prefix}}assets/swfobject.js"></script>` in started.html and components.html